### PR TITLE
add missing types to some arguments in third_party/3

### PR DESCRIPTION
**Pull request by JelleZijlstra**
Found these with stublint.

### CODE REVIEW

File: third_party/3/itsdangerous.pyi [hunk 1]
```diff
@@ -15,6 +15,7 @@ _bytes_like = Union[bytearray, bytes]
 _str_like = Union[str, bytes]
 _can_become_bytes = Union[str, bytes, bytearray]
 _comparable_bytes = TypeVar('_comparable_bytes', str, _bytes_like)
+_serializer = Any  # must be an object that has "dumps" and "loads" attributes (e.g. the json module)
```
Comment (JelleZijlstra) [inline third_party/3/itsdangerous.pyi]: @ilevkivskyi is this a use case that protocols can/should support?

**Comment by ambv** [comment]
> stublint

We are running flake8 with a few plugins already. The relevant one is flake8-pyi here which enables forward reference support. Do you have this configured, too? It found name errors in your PR: https://travis-ci.org/python/typeshed/jobs/210813094

I'd accept PRs to extend flake8-pyi with the kind of checks you're making in stublint. Would be good to run this on our CI!

**Comment by JelleZijlstra** [comment]
Oops, fixing the missing names (running local tests first this time).

stublint is at https://github.com/JelleZijlstra/stublint. Maybe it would be better written as a flake8 plugin; let me think about it.

File: third_party/3/itsdangerous.pyi (hunk 1 above)
Comment (ilevkivskyi) [inline third_party/3/itsdangerous.pyi]: Exactly!

**Pull request merged**